src/parquet/util/input.h

```diff
@@ -55,6 +55,9 @@ class PARQUET_EXPORT RandomAccessSource {
   int64_t size_;
 };
 
+// ----------------------------------------------------------------------
+// SAMPLE implementations of RandomAccessSource, for reading files from local disk
```

Comment:
"Implementations of RandomAccessSource used for testing and internal CLI tools. May not be robust for general production use."